srcpkgs/sview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+archs="i686 x86_64"
```

Comment:
Don't set archs, let it fail until figured out. Will be easier for us to look at error.

Reply:
I'm finding my experience with `./xbps-src -a <arch> pkg sview` quite different from what I see in TravisCI. Am I doing something wrong? Is there a way to implement the same build environment as you are using for TravisCI? I'd like to avoid pushing bunch of testing commits.

Reply:
For example i686 build is failing in chroot while passing compilation test in TravisCI, arm\aarch and mips compilations also fails quite differently to what Travis shows, however the template is the same across both, hence the question.

Reply:
Are you using a i686 masterdir to build it for i686?

Can you show the errors so I can try to compare them?

Reply:
I'd make sure your local repo is up to date, tho.